web3/_utils/module_testing/eth_module.py

```diff
@@ -741,6 +742,38 @@ def test_eth_call_with_0_result(
         result = web3.codec.decode_single('uint256', call_result)
         assert result == 0
 
+    def test_eth_call_revert_with_msg(
+        self,
+        web3: "Web3",
+        revert_contract: "Contract",
+        unlocked_account: ChecksumAddress,
+    ) -> None:
+        with pytest.raises(SolidityError, match='execution reverted'):
```

Comment:
Can we match a more specific part of the message here and the next test (like `execution reverted: not allowed to monitor` in `test_method_formatters.py`)?